frontend/src/services/your_clubs_handler_service/your_clubs_handler_service.ts

```diff
@@ -8,6 +8,13 @@ export class ErrorLoadingClubs extends Error {
   }
 }
 
+/* Error that occurs when an attempt to create a club fails. */
+export class ErrorCreatingClub extends Error {
+  constructor(readonly club: ClubProps) {
+    super(`Error occurred while creating '${club.name}' club.`);
```

Comment:
Nit: Do you think we could restructure the error messages that include 'club' at the end of them? It might be confusing if a club's name is, for example, "Moralities Book Club" and the error message that shows up in creating the club is "Error occurred while creating 'Moralities Book Club' club." Maybe we could change the message to be something like "Error occurred while creating the club 'Moralities Book Club'" (though this sounds kinda funny too, so it's up to you but something to think about in terms of user experience).